crates/bitwarden-ssh/src/import.rs

```diff
@@ -0,0 +1,214 @@
+use ed25519;
+use pkcs8::{der::Decode, pkcs5, DecodePrivateKey, PrivateKeyInfo, SecretDocument};
+use ssh_key::private::{Ed25519Keypair, RsaKeypair};
+
+use crate::{error::SshKeyImportError, SshKey};
+
+const PKCS1_LABEL: &str = "RSA PRIVATE KEY";
+const PKCS8_UNENCRYPTED_LABEL: &str = "PRIVATE KEY";
+const PKCS8_ENCRYPTED_LABEL: &str = "ENCRYPTED PRIVATE KEY";
+const OPENSSH_LABEL: &str = "OPENSSH PRIVATE KEY";
+
+pub fn import_key(
```

Comment:
⛏️ I'd like to second Oscar's feedback about documentation. If something is `pub`, it should be documented.

🎨 Fun fact! [Tests provided in rust's documentation are executable!](https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html). This is of particular use when describing error cases that you expect the consuming code to handle.